test/validators.js

```diff
@@ -5174,12 +5174,14 @@ describe('Validators', () => {
       {
         locale: 'pt-BR',
         valid: [
-          '+55-12-996551215',
```

Comment:
Could you explain this regression to L5182? Replacing `-` with _space_ separators...

Reply:
@profnandaa dash is generally only used before the last 4 digits of a Brazilian phone number (https://en.wikipedia.org/wiki/Telephone_numbers_in_Brazil#Inbound). Though it's also possible to have a dash between the country code, they are either not used at all or extremely rare (countrywide). Also, by testing it with some dialers (including: Hangouts Dialer, Android Dialer and AWS SNS), they'd replace dash with spaces/nothing. 

+55-11-99123-4567 would become +55 11 99123-4567 or +5511991234567

Following patterns are the most used countrywide:
+cc aa xxxx-xxxx (landline & mobile)
+cc aa 9xxxx-xxxx (mobile only)
(aa) xxxx-xxxx (landline & mobile)
(aa) 9xxxx-xxxx (mobile only)

What do you think?

I mean, it also depends on the pattern that has been used by validatorjs for other locales.

Also I want to point out that a removed carrier's codes from the validation as they might lead to duplicates as (+55 15 11 99123-4567 and +55 21 11 99123-4567 for the purpose of comparing them they are different, but they are actually the same number -> +55 11 99123-4567 [no carrier code]).

Reply:
Thanks for the explanation, makes sense.